shell/src/main/java/alluxio/cli/fsadmin/pathconf/ListCommand.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.pathconf;
+
+import alluxio.cli.CommandUtils;
+import alluxio.cli.fsadmin.command.AbstractFsAdminCommand;
+import alluxio.cli.fsadmin.command.Context;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.exception.AlluxioException;
+import alluxio.exception.status.InvalidArgumentException;
+import alluxio.wire.Configuration;
+
+import org.apache.commons.cli.CommandLine;
+
+import java.io.IOException;
+
+/**
+ * Lists paths that have path level configuration.
+ */
+public final class ListCommand extends AbstractFsAdminCommand {
```

Comment:
does this need to extend `AbstractFsAdminCommand`? Usually things extending that can be called directly like `fsadmin list`, not via subcommand. See `ReportCommand` and its subcommands for example

Reply:
by extending `AbstractFsAdminCommand`, the subcommand's interface and implementation can have the same structure as other commands. From a user's perspective, a subcommand is also a fsadmin command.